supervisord.conf

```diff
@@ -0,0 +1,8 @@
+[supervisord]
+nodaemon=true
+
+[program:gulp]
+command=/bin/bash -c "/crowbar-ui/node_modules/gulp/bin/gulp.js"
```

Comment:
ok, i think autorestart is default (but i need to check). at least when i killed some process inside, it respawned automatically